src/encrypted_table/mod.rs

```diff
@@ -413,6 +419,22 @@ impl EncryptedTable<Dynamo> {
         })
     }
 
+    pub async fn init_with_zk_config(
```

Comment:
Same suggestion here:  mind spelling out `zerokms` instead of abbreviating?
```suggestion
    pub async fn init_with_zerokms_config(
```